plugins/eddn.py

```diff
@@ -35,121 +46,147 @@
 this.marketId = None
 this.commodities = this.outfitting = this.shipyard = None
 
+HORIZ_SKU = 'ELITE_HORIZONS_V_PLANETARY_LANDINGS'
+
 
-class EDDN(object):
+# TODO:
+# 1, this does a bunch of magic checks for whether or not the given commander has horizons. We can get that directly
```

Comment:
I wouldn't just assume that the CAPI indication can be 100% relied upon.  Heck I see reports about the *game* sometimes being confused as to if a Cmdr has Horizons or not.  So I'd not change of these checks unless they're provably broken.